code/game/machinery/ecto_sniffer.dm

```diff
@@ -34,12 +40,13 @@
 /obj/machinery/ecto_sniffer/proc/activate(mob/activator)
 	flick("ecto_sniffer_flick", src)
 	playsound(loc, 'sound/machines/ectoscope_beep.ogg', 25)
-	visible_message("<span class='notice'>[src] beeps, detecting ectoplasm! There may be additional positronic brain matrixes available!</span>")
+	var/msg = "[src] beeps, detecting ectoplasm! There may be additional positronic brain matrices available!"
```

Comment:
`var/msg = "[src] beeps, detecting ectoplasm! There may be additional positronic brain matrices available!"`

Typo

Reply:
nvm someone requested that lol